### PR TITLE
Hou updates

### PR DESCRIPTION
**Pull request by kchia**
To-Do's:
- [ ] Add back-end tests
- [ ] Add front-end tests

### CODE REVIEW

File: back-end/src/reservations/reservations.service.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ function create(newReservation) {
 function list(date) {
   return knex(tableName)
     .where("reservation_date", date)
-    .andWhereNot("status", "finished")
+    .whereNotIn("status", ["finished", "cancelled"])
```
Comment (kchia) [inline back-end/src/reservations/reservations.service.js:136]: @dalelotts `andWhereNot` was not working.

Comment (dalelotts) [reply back-end/src/reservations/reservations.service.js:136]: Was a test failing? or did you observe this? or both?

Comment (dalelotts) [reply back-end/src/reservations/reservations.service.js:136]: Trying to figure out if I screwed up the test or if it was something else?

Comment (dalelotts) [reply back-end/src/reservations/reservations.service.js:136]: @kchia ^^

Comment (kchia) [reply back-end/src/reservations/reservations.service.js:136]: I observed it.

**Pull request merged**